lua/luasnip/init.lua

```diff
@@ -510,21 +513,21 @@ local function unlink_current_if_deleted()
 	local snippet = node.parent.snippet
 	local ok, snip_begin_pos, snip_end_pos =
 		pcall(snippet.mark.pos_begin_end_raw, snippet.mark)
-	-- stylua: ignore
```

Comment:
Oh, this section is ignored by stylua, could you manually revert it?
(same a few lines further down)

Reply:
Oh oops, I think I formatted it with sumneko instead of stylua. The dangers of having multiple formatters lol. Fixed in https://github.com/L3MON4D3/LuaSnip/pull/652/commits/2c1fda2ad554f6c674cd72df704fe58a3e7d237b hopefully.

Reply:
Nice, that did it :+1: